.github/workflows/vectorlint.yml

```diff
@@ -0,0 +1,27 @@
+name: Vector Lint
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '15 */12 * * *'
+
+jobs:
+  build:
+    name: Lint vectors/schemas
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v5
+
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Format
+        run: go fmt ./...
+
+      - name: Build
+        run: go build -v ./...
+
+      - name: Run vectorlint
+        run: ./vectorlint
```

Comment:
These two steps can be just `go run ./tools/vectorlint` without creating an untracked binary file.

Reply:
Done.